charts/portal/values-sample-redeploy.yaml

```diff
@@ -0,0 +1,62 @@
+global:
```

Comment:
Why did we create a new file here?  It has internal endpoints as well, which we don't want exposed in our codebase

Reply:
i'm just showing example of the values to overwrite. this file will be removed before merge